AtomAnu/COMP0036_EPL_Prediction
# Add a script to extract data from the training data file

Description of Work:
A script `extract_data.py` to extract data from the training file `epl-training.csv` has been added. Apart from extracting the data, it should locate any categorical features (columns) and compute all the possibles values that each feature could take.

Fixes: #2 

Testing Instructions:
1. Clone the branch `add_extract_data_script` and run `extract_data.py`. Verify the results.
